modules/@apostrophecms/video-widget/ui/public/video.js

```diff
@@ -54,6 +54,10 @@ apos.util.widgetPlayers['@apostrophecms/video'] = {
         if (result.width && result.height) {
           inner.style.width = '100%';
           inner.style.height = ((result.height / result.width) * inner.offsetWidth) + 'px';
+          // If we need to iniitally size the video, also resize it on window resize.
+          window.addEventListener('resize', function() {
+            inner.style.height = ((result.height / result.width) * inner.offsetWidth) + 'px';
```

Comment:
This handler should check that `inner` is still in the DOM and if not it should unhook the listener and politely return.

Reply:
Also, suggest factoring the logic into a little function that can be called twice rather than repeated.

Reply:
Looks like `document.contains` works at least back to IE11.